src/language-yaml/utils.js

```diff
@@ -69,6 +69,27 @@ function isNextLineEmpty(node, text) {
   return false;
 }
 
+function isPreviousLineEmpty(node, text) {
```

Comment:
Why don't you use `isPreviousLineEmpty` from `common/util`?

Reply:
Don't know that.

Reply:
Fixed